force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -270,14 +325,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return The position of the word in the list as a string or -1 if the word was not found.
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
-        // The variable to store the index of the word
-
-        // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+       
+        Integer stringPosition = 0;
+        for (Integer i = 0; i <= words.size(); i++) {
+            if (words.equals(wordToFind)) {
+                break;
```

Comment:
Break will exit the loop entirely and you'll never assign a value to stringPosition, so make sure you assign the value before breaking